### DISCUSSION TITLE
Thank you from Nix!

### DISCUSSION THREAD

**Discussion by asomers** [cirruslabs/cirrus-ci-docs#743] [General]
Thanks for the great service you provide to the open source community.  I just converted my project's CI from Travis to Cirrus, and cut test times down from 6+ hours to ~ 10 minutes.  So refreshing!
https://github.com/nix-rust/nix/runs/1554711584

**Comment by fkorotkov** [comment]
Wow! That is impressive! Congrats on such an achievement! 🙌
